requirements.txt

```diff
@@ -38,9 +38,9 @@ Flask-RESTful==0.3.9
 furl==2.1.0
 futures==3.1.1
 geoip2==3.0.0
-gevent==21.8.0
+gevent
 greenlet==1.1.2
-grpcio==1.46.3
+grpcio
```

Comment:
we need pinned versions here - did this cause any issues?

Reply:
@flavianmissi @HammerMeetNail  our intention was to use latest versions. We are currently using pre-built wheel files for ppc64le arch under official IBM repositories. Since we have pre-built binaries for grpcio v1.46.3 and gevent 21.8.0 we can continue using same versions. I have updated the code accordingly. Can you please re-review?